docs/conf.py

```diff
@@ -906,7 +906,6 @@ def filter_ignore(record: logging.LogRecord) -> bool:
             "spec": OPENAPI_FILE.as_posix(),
             "opts": {
                 "hide-hostname": True,
-                "no-auto-auth": True,
```

Comment:
This change looks unrelated.

Reply:
They are related. Setting `True` to `no-auto-auth` disable/remove the authentication section in the API documentation. Here we want to enable it